pkg/api/meta/meta.go

```diff
@@ -28,6 +29,40 @@ import (
 	"github.com/golang/glog"
 )
 
+func ListAccessor(obj interface{}) (List, error) {
+	if oi, ok := obj.(ListMetaAccessor); ok {
+		if om := oi.GetListMeta(); om != nil {
+			return om, nil
+		}
+	}
+	// we may get passed an object that is directly portable to List
+	if oi, ok := obj.(List); ok {
```

Comment:
Because ListMeta is an embedded field, so if I check this first, the entire list object will be return. I want to return only the ListMeta.